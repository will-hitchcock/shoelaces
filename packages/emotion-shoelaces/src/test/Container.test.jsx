import React from 'react';
import { shallow } from 'enzyme';
import { Container } from '..';

describe('<Container />', () => {
  it('should render a <Container />', () => {
    expect(shallow(<Container />)).toMatchSnapshot();
  });
});
