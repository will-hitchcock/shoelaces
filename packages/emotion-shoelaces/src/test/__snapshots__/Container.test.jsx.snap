// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> should render a <Container /> 1`] = `
<styled.div
  _css={
    Object {
      "@media (min-width: 105em)": Object {
        "width": "99em",
      },
      "@media (min-width: 36em)": Object {
        "width": "34em",
      },
      "@media (min-width: 48em)": Object {
        "width": "45em",
      },
      "@media (min-width: 62em)": Object {
        "width": "58em",
      },
      "@media (min-width: 75em)": Object {
        "width": "70em",
      },
      "marginLeft": "auto",
      "marginRight": "auto",
    }
  }
/>
`;
